themes/default/views/editor/objects/screen_html.php

```diff
@@ -82,4 +82,56 @@
 
 	<div class="editorBottomPadding"><!-- empty --></div>
 	
-	<?php print caSetupEditorScreenOverlays($this->request, $t_object, $va_bundle_list); ?>
\ No newline at end of file
+	<?php print caSetupEditorScreenOverlays($this->request, $t_object, $va_bundle_list); ?>
+	<script type="text/javascript">
+		(function(initSplitSuffix) {
+
+			initSplitSuffix(window.jQuery);
+
+		}(function($) {
+
+				$(function () {
+					var reg = /^\s*\.?\s?(\d*)\s*(.*)\s*$/,
+						parse = function (val) {
+							if (val && typeof val === 'string') {
+								var parts = val.match(reg);
+								parts.shift();
+								return parts;
+							}
+						};
+					$('#idno_accession_series_and_parts').each(function () {
+						var $dot = $('<span class="idno_dot">.</span>').hide(),
+							$series = $('<input type="number" class="idno_accession_series" title="Enter series number" step="1" min="1"/>'),
+							$suffix = $('<input type="text" class="idno_accession_suffix" size="10"/>'),
```

Comment:
The built in CA fields use `@size`, but I'm using an `<input type='number' />` which doesn't support `@size`, so I had to do this.